### PR TITLE
added message model

### PR DESCRIPTION
**Pull request by KaitCrawford**
review and merge https://github.com/praekelt/mentorship/pull/3 first

### CODE REVIEW

**Comment by KaitCrawford** [comment]
@miltontony ready for review

**Comment by miltontony** [comment]
Need to merge #3 first, but looks good from what I can see so far. I'll only +1 after the merge

**Comment by KaitCrawford** [comment]
@miltontony previous PR merged

File: mentorship/mentor/views.py [hunk 1]
```diff
@@ -15,10 +15,31 @@ def home(request):
...
+    if 'message_id' in request.session:
+        context.update({
+            'message_id': request.session['message_id']
+        })
+        del request.session['message_id']
     return render(request, 'mentor/home.html', context)
 
 
 @login_required(login_url='/login/')
+def send_message(request):
```
Comment (miltontony) [inline mentorship/mentor/views.py]: Similar to another content I made - it's probably better to use a `FormView` here.

**Comment by KaitCrawford** [comment]
@miltontony ready for rereview

**Comment by miltontony** [comment]
:+1: :cake:

**Pull request merged**